pkg/security/common/containerutils/utils.go

```diff
@@ -7,42 +7,87 @@
 package containerutils
 
 import (
+	"path/filepath"
 	"regexp"
 	"strings"
 )
 
+// CGroup managers
+const (
+	CGroupManagerDocker  uint64 = 1 << 0
+	CGroupManagerCRIO    uint64 = 2 << 1
+	CGroupManagerPodman  uint64 = 3 << 2
+	CGroupManagerCRI     uint64 = 4 << 3
+	CGroupManagerSystemd uint64 = 5 << 4
+)
```

Comment:
Could it be possible to have a dedicated type here? It could help to make the link between those values and the different int64 struct fields where they are used (model/model_unix/etc)